### ISSUE TITLE
LightGBM ARCH

### ISSUE THREAD

**Issue by waudinio27** [SaremS/sample_notebooks#1]
Dear Sarem!

Your blog and the notebook are amazing! Do you also work with particle filters for the stock market?

**Comment by SaremS** [comment]
Thanks! Unfortunately, no. I did some experiments with Stein gradient descent some time ago (https://arxiv.org/abs/1608.04471) but not much classical particle filters yet.

**Comment by waudinio27** [comment]
Still it is great work! I started to follow you on Medium :-D Did you try your model on a demo account? I think it has potential! I think a particle filter could be usefull for volatility forecasting, but it is a project more for the future.

**Comment by SaremS** [comment]
Not yet. I’m currently working on something else for trading. Can’t say much though as I’m NDA‘ed.

But I always liked the idea of GARCH for volatility arbitrage on options. Will definitely try that out some time.
